core/controllers/base_test.py

```diff
@@ -526,6 +527,17 @@ def test_redirect_oppia_test_server(self):
         self.assertEqual(
             response.headers['Location'], 'https://oppiatestserver.appspot.com')
 
+    def test_splash_redirect(self):
+        # Tests that the old '/splash' URL is redirected to '/'.
+
+        response = self.get_html_response('/splash', expected_status_int=302)
+        response_path = response.headers['location']
+        # The following asserts ensure that the the URL ends with `/` and the
```

Comment:
Have a consistent character for quotation, use `'` (as in line 531.)

Reply:
Done